### ISSUE TITLE
[Discussion] accessibility changes to breadcrumbs

### ISSUE THREAD

**Issue by RayBB** [internetarchive/openlibrary#4949]
### Question

What, if any, changes should be made to breadcrumbs to improve accessibility? 

### Additional context
I am currently working on #4902 (accessibility for the [individual list page](https://openlibrary.org/people/mekBot/lists/OL103955L/Childhood_Favorites)).

When using the screenreader (which is not something I normally do), realized that there is a possibility for the username portion of the breadcrumbs to be confusing.
This is because:
1. The username is the first link a user will see in the body of the list page (or will be after #4939 )
2. There is no label o indication for screen readers that it is a username
3. The username could be deceiving (like "borrow_a_book")

I was quite puzzled by this and trying tried searching for discussions about this without finding much.
I posted on stack overflow ([Should aria-label be used to describe user generated content like username links?](https://stackoverflow.com/q/66864742/620699)) and was lucky to get a fantastic response.

[The response](https://stackoverflow.com/a/66877060/620699) suggested that, in this case, the breadcrumbs do make sense but they could be improved by putting them into an unordered list.

There is perhaps a bigger discussion about other places usernames are shown but for now I think this ticket should focus on the breadcrumbs.

The response makes sense to me but I don't know much about a11y yet so wanted to open the floor to discussion. 

<details>
  <summary>Screenshot of breadcrumbs</summary>
  
  ![image](https://user-images.githubusercontent.com/921217/113080006-d40ebe00-9171-11eb-9a1f-0b0f5c5e0cb3.png)
  
</details>

Example page: https://openlibrary.org/people/mekBot/lists/OL103955L/Childhood_Favorites

### Issue resolution criteria
Consensus? 

### Stakeholders
@bpmcneilly

**Comment by mekarpeles** [comment]
We have a new breadcrumbs method now which needs work (on My Books) but I believe there are issues for it, so this one can be closed.
